db/seeders/20240505140420-members.ts

```diff
@@ -0,0 +1,54 @@
+"use strict";
+
+/** @type {import('sequelize-cli').Migration} */
+
+import { QueryInterface, DataTypes } from "sequelize";
+module.exports = {
+  async up(queryInterface: QueryInterface, Sequelize: typeof DataTypes) {
+    await queryInterface.bulkInsert("members", [
+      {
+        full_name: "Owner",
```

Comment:
In the next commit, do change the names of the seeded members to avoid confusion with the roles. Thanks!